code/game/objects/items/weapons/tools/_tools.dm

```diff
@@ -217,31 +217,29 @@
 				spawn(100)
 					H.disabilities &= ~NEARSIGHTED
 
-//Prosthetic repair
-/obj/item/weapon/tool/attack(var/mob/living/carbon/human/H, var/mob/living/user)
 
-	if(get_tool_type(user, list(QUALITY_WELDING)))
-		if(ishuman(H))
-			var/obj/item/organ/external/S = H.organs_by_name[user.targeted_organ]
+/obj/item/weapon/tool/attack(mob/living/M, mob/living/user, var/target_zone)
```

Comment:
this section seems a bit out of scope, what's this all about?

Reply:
The prosthetic repair was not resolving the attack properly and under certain circumstances just returning without feedback so I fixed the conditions and returns.